src/templates/author-template.tsx

```diff
@@ -0,0 +1,66 @@
+import { graphql, Link } from "gatsby"
+import * as _ from "lodash"
+import * as React from "react"
+import Layout from "../components/Layout"
+
+interface AuthorTemplateProp {
+  data: {
+    allMarkdownRemark: {
+      nodes: [
+        {
+          frontmatter: {
+            date: string
+            author: string[]
+            title: string
+          }
+        }
+      ]
+    }
+  }
+}
+
+const AuthorPageTemplate: React.FC<AuthorTemplateProp> = ({ data }) => {
+  const allNodes = data.allMarkdownRemark.nodes
+  const [name, setName] = React.useState(undefined)
+
+  return (
+    <div>
```

Comment:
Is this `div` needed?

Reply:
@damassi 
You mean I should use fragment ` <> </>` instead of using `<div>`. Right now, I only have one, but it is not finalized. In the future, If I planned to rerender more than one, should I use the wrapper div or using fragments?